e2e/browser/test-app/components/appContainer/index.tsx

```diff
@@ -28,7 +28,7 @@ import {
 } from "@inrupt/solid-client-authn-browser";
 import AuthenticatedFetch from "../authenticatedFetch";
 
-const REDIRECT_URL = window.location.href;
+const REDIRECT_URL = "http://localhost:3001";
```

Comment:
I know this is a conversation we've been having, but I still believe that using `window.location.href` is a better pattern than hardcoding it. 

Two thoughts that have come from building apps and using both approaches:
- reusability: If the redirect url is being set dynamically, you can easily reuse components without having to hardcode the redirect URL. This is especially helpful in the case of routing and deeplinking where you can define which routes should be allowed to redirect in your client identifier document but you don't need to explicitly call it out in the code. 
- separating deployment details from code: when we hardcode the redirect url in the code, we have to know where our application is being hosted. This to me should be as separate from the code itself and should be handled in our CI/CD process or package file or env file and not in the source code itself.

We may run into issues if a `window.location.href` in our application is not in our client identifier document and the redirect will fail, but I think instead of calling this an issue or a failure point, what is more beneficial is to understand what happened and either add a route to the client identifier document or update how the application is deployed. 


*Note this is not a blocking comment as I don't want to block this PR before we wrap up the conversation*